lib/evmone/eof.cpp

```diff
@@ -623,53 +624,70 @@ std::variant<EOFValidationError, int32_t> validate_max_stack_height(
     return max_stack_height_it->max;
 }
 
-std::variant<EOF1Header, EOFValidationError> validate_eof1(  // NOLINT(misc-no-recursion)
-    evmc_revision rev, bytes_view container) noexcept
+std::variant<EOF1Header, EOFValidationError> validate_eof1(
+    evmc_revision rev, bytes_view cont) noexcept
```

Comment:
If we change `container` to `cont` I would make it in the whole function. So also `subcontainer` to `subcont`

Reply:
It was to disambiguate from `container` inside the loop, but a bad name, will change to `main_container`.

Reply:
It was to disambiguate from `container` inside the loop, but a bad name, will change to `main_container`.